utils/Pipfile

```diff
@@ -9,6 +9,7 @@ verify_ssl = true
 pandas = "*"
 argparse = "*"
 math = "*"
```

Comment:
Oops, forgot about the pipfile. I didn't change this file because I didn't know what to do with it, since we are moving on to poetry. Should I delete this file instead?